content/source/docs/cloud/users-teams-organizations/single-sign-on/azure.html.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: "cloud"
+page_title: "Azure - Singlge Sign-On - Terraform Cloud and Terraform Enterprise"
+---
+
+-> **Note:** Single sign-on is a paid feature, available as part of the **Business** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
+
+# Single Sign-on: Azure
+
+The Azure SSO integration currently supports the following SAML features:
+
+- Service Provider (SP)-initiated SSO
+- Identity Provider (IdP)-initiated SSO
+- Just-in-Time Provisioning
+
+## Configuration (Azure ) Part 1:a
+
+-> **Note:** Azure is rolling out a new interface for setting up Enterprise Applications. If you are using the new Azure UI, see the section below, headed as `Configuration (Azure ) 1:b`
+
+1. In your desired Azure AD Directory, navigate to the Enterprise Applications page and create a "+ New Application".
+2. Select the non-gallery application option.
+3. Give your application a name and click "Add."
+
+## Configuration (Azure) Part 1:b
+
+1. In your desired Azure AD Directory, navigate to the Enterprise Applications page and create a "+ New Application".
+2. locate and select the "+ Create your own appliction" option.
+3. In the resulting sidebar, name your application, and select the option, "Integrate any other application you don't find in the gallery" and click "Create" at the bottom of the page.
+
+## Configuration (Azure) Part 2
+
+1. Under the "Getting Started" section, select "Set up single sign on."
+2. Select the SAML option
+3. You should now see a 5 part set up list. Ignore step 1 for now, we will come back to that.
+4. Edit step 2, "User Attributes & Claims"
+    1. Select the required Name ID claim.  Ensure that its name identifier format is "Email address".
+    * If you plan on using Terraform Clouds SAML team management, add a new claim.
```

Comment:
I feel we should skip this from the initial setup steps. We should focus on getting them using SSO as quickly as possible and if they wish to manage teams it can be in additional documentation.